test/app-forms.spec.js

```diff
@@ -183,7 +183,7 @@ describe('forms that have caused bugs', () => {
     expect(twoTels.report).to.nested.deep.include({
       form: 'tel',
```

Comment:
i ran some tests with `headless: false` passed to Harness constructor. I'm seeing them fail because harness.consoleErrors has the following content

```
Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: file:///home/kenn/cht-conf-test-harness/dist/cht-core-4-6/NotoSans-Bold.ttf
```

Time to look into that?

Reply:
Okay, I was able to recreate this by throttling the network in chromium while running the tests.  (Just running normally with `headless: false` works okay for me....)

Seems like this is standard expected behavior when trying to load fonts on what Chromium considers to be a "Slow network".  I have found two possible solutions here:

- Add the error to the list of ignored errors and just not worry about it. :shrug:  (Functionality should not be effected at all by this error.)
- Update the webpack config to not include the font files at all.  I tested this and it did cause the error to go away.  One upside of this is that we would not have all those extra font files committed into `dist/cht-core-4-6`.  The missing fonts is not really a big deal for the automated tests with the harness, but it will affect forms displayed in the Project Explorer (e.g. the font-awesome icons will not be displayed...)

I am going to update the code according to the first option, but I can roll it back out if you would prefer the second (or if you have a better idea here...)!

Reply:
1 seems great. kudos that u actually did so much to try 2...